sdk/search/search-documents/review/search-documents.api.md

```diff
@@ -474,13 +489,33 @@ export type IndexerStatus = 'unknown' | 'error' | 'running';
 // @public
 export interface IndexingParameters {
     batchSize?: number;
-    configuration?: {
-        [propertyName: string]: any;
-    };
+    // (undocumented)
+    configuration?: IndexingParametersConfiguration;
     maxFailedItems?: number;
     maxFailedItemsPerBatch?: number;
 }
 
+// @public
+export interface IndexingParametersConfiguration {
+    [property: string]: any;
```

Comment:
this is unfortunate - I'm guessing the swagger has `additionalProperties: true` for some reason?

Reply:
Yes. They have added here: https://github.com/Azure/azure-rest-api-specs/pull/10349/files#diff-54a85d8309f1ab82d2da096e6e430b9fR5870 the value:
```
"additionalProperties": {
    "type": "object"
}
```